### ISSUE TITLE
Set metadata schema to JSON for top level

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1924]
And give an example of adding some

**Comment by jeromekelleher** [comment]
Probably not worth bothering with this until we have some easy examples of using the tskit Metadata schema builder to add some metadata in a way that we'd approve of (i.e., with an explicit schema).

Just dumping a bunch of stuff into persmissive_json is something we'll regard as an anti-pattern, so there's no point in encouraging people to do it.

**Comment by benjeffery** [comment]
Agreed, the metadata API shouldn't be too much work. Maybe we should move it to directly after C1.0?
